### PR TITLE
Added copy_target and open_browser config options (also a config file bugfix)

### PR DESCRIPTION
**Pull request by Naishaline**
I added the ability to specify a "copy_target" in the config. If it's set to `image` it'll copy the raw image data to your clipboard, `direct` will copy the direct image url from Gyazo. If unspecified or anything else, it'll just copy the Gyazo link like normal.

I also added an "open_browser" option that will let you decide if you want to open the Gyazo page after you take a screenshot.

Finally, if someone has a config file with no content in it, it causes crashes and errors. I added a check to see if the file is empty, and if it is I replace the config variable with `{}`.

If you have any changes or questions let me know!

### CODE REVIEW

**Review by yuiseki — APPROVED** [review]
Thank you for Pull Request!

I've checked this PR works with no probrem.

Check point:
edit `~/.gyazo.config.yml`
- [x] `copy_target` option
  - [x] when value is empty, copy gyazo URL to clipboard
  - [x] when value is `image`, copy image binary to clipboard
  - [x] when value is `direct`, copy direct image URL to clipboard
- [x] `open_browser` option
  - [x] when value is empty, open browser
  - [x] when value is `false`, dont open browser
  - [x] when value is `true`, open browser

File: src/gyazo.rb [hunk 1]
```diff
@@ -138,3 +157,5 @@
     File.open(idfile,"w").print(newid)
   end
 }
+
+File.delete(tmpfile)
```
Comment (pastak) [inline src/gyazo.rb:161]: add newline at the end
```suggestion
File.delete(tmpfile)

```

File: src/gyazo.rb [hunk 2]
```diff
@@ -21,6 +26,13 @@
 ua = config['ua'] || 'Gyazo/1.3.1'
 http_port = config['http_port'] || 443
 use_ssl = config['use_ssl'] == nil ? 'true' : config['use_ssl']
+copy_target = config['copy_target'] || 'link'
+#False would trigger the || condition if using the syntax above
+if config['open_browser'] != nil
+  open_browser = config['open_browser']
+else
+  open_browser = true
+end
```
Comment (pastak) [inline src/gyazo.rb:35]: It looks good for default value
```suggestion
open_browser = true
if config['open_browser'] != nil
  open_browser = config['open_browser']
end
```

**Comment by pastak** [comment]
@Naishaline Thanks for your good works! I add some suggestion comments. plz check it.

**Comment by Naishaline** [comment]
I updated to match with your suggestions! Good call on the open_browser one.

**Comment by pastak** [comment]
Thank you for your contribution! We will release it soon in next version.

**Pull request merged**